.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
```

Comment:
what do you mean?

Reply:
I use PhpStorm, and it saves the setting project in the directory .idea.
It should not be in the repository because it is my personal settings for my IDE. To not accidentally add it into the commit, I added it in the git ignore files.

Reply:
I understand,it is from your IDE then not mine